src/browser/components/Tabs/Tabs.js

```diff
@@ -60,4 +68,22 @@ export default class Tabs extends React.Component {
       this.props.onClose(sessionId);
     };
   }
+
+  handleShortcuts = (event) => {
+    // Command + w (close current tab or client)
+    if (event.metaKey && event.key === 'w') {
```

Comment:
I could be mistaken, but I believe `metaKey` is only valid on macOS.  We may need to conditionally use `metaKey` for macOS and check for `ctrlKey` on Windows/Linux.

Reply:
Good point, I'll add it.